test/arrays/diten/replaceArrayAccess/zipperModify.chpl

```diff
@@ -0,0 +1,12 @@
+var A, B: [1..4] int = 1..4;
+
+proc f(i) return i;
+
+for (a, i) in zip(A, 1..4) {
+  A[i] = A[i] * A[i];
```

Comment:
This test doesn't match the pattern this is looking for, so it currently doesn't apply.